libraries/SPI/src/SPI.h

```diff
@@ -1,18 +1,18 @@
 #ifndef SPI_H
 #define SPI_H
 
-#include <Arduino.h>
+#include "Arduino.h"
+#include "HardwareSPI.h"
```

Comment:
here, we shall use the "api/HardwareSPI.h"? According to this conversation: https://github.com/arduino/ArduinoCore-API/issues/37#issuecomment-2718229952

Reply:
Perfect! I think it will still work as we added the depecrated includes. But so that we already have it right.